datadog/tests/cassettes/TestAccDatadogLogsMetric_Basic.yaml

```diff
@@ -3,7 +3,7 @@ version: 1
 interactions:
 - request:
     body: |
-      {"data":{"attributes":{"compute":{"aggregation_type":"distribution","path":"@duration"},"filter":{"query":"service:test"},"group_by":[{"path":"@my.status","tag_name":"status"},{"path":"service","tag_name":"service"}]},"id":"tf_TestAccDatadogLogsMetric_Basic_local_1615587158","type":"logs_metrics"}}
+      {"data":{"attributes":{"compute":{"aggregation_type":"distribution","include_percentiles":false,"path":"@duration"},"filter":{"query":"service:test"},"group_by":[{"path":"@my.status","tag_name":"status"},{"path":"service","tag_name":"service"}]},"id":"tf_TestAccDatadogLogsMetric_Basic_local_1669226251","type":"logs_metrics"}}
```

Comment:
Could we add a test for the `PATCH` with a change of the `include_percentiles`? There is similar test [here](https://github.com/DataDog/terraform-provider-datadog/blob/e15155c75c87900d52f09bc467afd2b6f4f09577/datadog/tests/cassettes/TestAccDatadogMetricTagConfiguration_Basic.yaml#L97) for standard metrics

Reply:
Added patch test case here 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/6bbb1e072a7354cd6d021cdff098740c5087f143)